packages/editor/src/components/widgets/code-editor/ScriptInput.css

```diff
@@ -16,3 +16,7 @@
   right: 0;
   margin: var(--size-2);
 }
+
+.input-badge-wrapper {
+  width: 100%;
+}
```

Comment:
why needed?

Reply:
As it's not set on the component, it's needed that the input uses the full width
if it's set on the component, it would lead to problems in the form-editor

Reply:
Then it just runs out the side :smile: 
![image](https://github.com/user-attachments/assets/13c61f6f-7e19-4830-a462-5ca5f0111e85)

Reply:
if you do it the same as it is on the input with calc(100% - 2 * paddings)?